tide_core.install

```diff
@@ -82,3 +85,24 @@ function tide_core_update_8097() {
     $editor->save();
   }
 }
+
+/**
+ * Setup TFA.
+ */
+function tide_core_update_8098() {
```

Comment:
@vincent-gao were we renumbering updates? Should this be `tide_core_update_10001()`?

Reply:
hey @anthony-malkoun , yes.  ideally, we should keep the update hook number consistent with the main version of Drupal.

Reply:
Updated